eslint.config.js

```diff
@@ -29,4 +29,14 @@ export default [
       },
     },
   },
-];
+
+  // Tests
+  {
+    files: ['**/*-test.js'],
+    languageOptions: {
+      globals: {
+        ...vitest.environments.env.globals,
+      },
+    },
+  },
```

Comment:
This ensures eslint recognizes vitest's globals. Most were detected by the mocha eslint plugin, but `beforeAll` and `afterAll` are not defined by mocha.

In the near future, we should update our eslint shared config to replace mocha's plugin with vitest one.